backend/x11/meson.build

```diff
@@ -16,14 +15,6 @@ foreach lib : x11_required
 	x11_libs += dep
 endforeach
 
-foreach lib : x11_optional
-	dep = dependency(lib, required: get_option(lib))
-	if dep.found()
-		x11_libs += dep
-		conf_data.set10('WLR_HAS_' + lib.underscorify().to_upper(), true)
```

Comment:
`WLR_HAS_XCB_XKB` needs to be removed from the toplevel `meson.build`

Reply:
Also from `include/wlr/config.h.in`

Reply:
Done